src/main/java/de/wirvsvirus/hack/service/InactivityCheckService.java

```diff
@@ -60,4 +101,49 @@ private void sendPushMessageInactiveUser(User inactiveUser,
         );
   }
 
+  private void sendPushMessageLazyUser(
+      final User lazyUser, final Group group) {
+
+    final List<String> otherUserNames =
+        onboardingRepository
+            .findOtherUsersInGroup(group.getGroupId(), lazyUser.getUserId()).stream()
+            .filter(User::hasName)
+            .map(User::getName)
+            .collect(Collectors.toList());
+    Collections.shuffle(otherUserNames);
+
+    onboardingRepository.findDevicesByUserId(lazyUser.getUserId())
+        .forEach(device -> pushNotificationService.sendMessage(
+            device.getFcmToken(), "Familiarise  - " + group.getGroupName(),
+            buildNoStatusUpdateString(otherUserNames),
+            Optional.empty(),
+            Optional.empty())
+        );
+  }
+
+  static String buildNoStatusUpdateString(final List<String> allOtherUserNames) {
+    final List<String> otherUserNames =
+        allOtherUserNames.stream()
+            .filter(name -> name.length() < 12)
+            .limit(3)
+            .collect(Collectors.toList());
+
+    final String pushText;
+    if (otherUserNames.isEmpty()) {
+      pushText = "Wie geht es dir gerade!";
+    } else if (otherUserNames.size() == 1) {
+      pushText = String.format("%s möchte wissen, wie es dir geht!",
+          Iterables.getOnlyElement(otherUserNames));
+    } else {
+      final String allButLast =
+          StreamEx.of(otherUserNames)
+              .limit(otherUserNames.size() - 1L)
+              .joining(", ");
+      final String last = Iterables.getLast(otherUserNames);
+      pushText = String.format("%s und %s möchten wissen, wie es dir geht!",
```

Comment:
Willst du echt alle Mitgliedernamen schicken? Ich finde, falls es mehr als 3 Leute sind, sollte man es formulieren in der Art: "Deine Fam-Group %s möchte wissen wie es dir geht!" oder sowas in der Art.

Reply:
das wurde schon auf 3 begrenzt